glean-core/src/lib.rs

```diff
@@ -941,6 +960,13 @@ pub fn glean_test_destroy_glean(clear_stores: bool, data_path: Option<String>) {
 
         dispatcher::reset_dispatcher();
 
+        // Only useful if Glean was fully initialized.
```

Comment:
Possible inconsistency: Is there a reason "Glean was fully initd" is determined differently here than at the top of shutdown()?

Reply:
Inconsistency indeed. `was_initialize_called` vs. "was called, did work and actually finish, including setting up a database"